script.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable max-classes-per-file */
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+/* eslint-disable max-classes-per-file */
```

Comment:
- [ ] [REQUIRED] Hello team, let's note that disabling eslint is not a good practice. Kindly enable eslint so it can check the standards of your Javascript code. 
- You currently have this error because you have more than one class in this file. The standard is having one class per file.

Below is an example of a class that can house all your functions.
```javascript
class Books {
  constructor() {
    this.books = [];
  }

  addBook(title, author) {
  }

  removeBook(bookID) {
  }


}
```

Reply:
![Tips-for project](https://user-images.githubusercontent.com/122881270/236119964-813995ed-c5ef-4af1-a54d-c2e9e9de77ce.PNG)
I highly appreciate of the reviewer insight and suggestion to instill the best practise in us.
However, one of the tips recommended by Microverse for this project based on challenges student faced with ESLinter error was to disable if it poses persistent errors.
Please, kindly advise. Thank you 🙏!

Reply:
Hello @SamuelTakyiBoateng , thank you for responding, however, this exemption is not on this particular milestone. Take a look at the second line of the screenshot you attached and see: "Make sure the data is preserved in the browser's memory by using localStorage". However, this milestone does not have anything to do with localStorage, hence you are looking at another milestone requirement and exemption. Here are the [milestone requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m2_plain_js_classes_v1_1.md) for better clarification.
Do let me know if you have any more questions or if you fill out the second opinion request in order to get an opinion from a Senior.

Thank you.

Reply:
Hello Sir,
Please, I'm very sorry🙏 if the my initial comments was not well received and really appreciate the kindness and  extra mile you take when reviewing codes (even with for my first experience with you). To be honest, we were just still confused and unable to implement the earlier suggestion for which we reached out.

Reply:
Hi, @SamuelTakyiBoateng Your comment was well received.
I am always available to help 😄